staticcheck.conf

```diff
@@ -0,0 +1 @@
+checks = ["all", "-ST1000", "-ST1005"]
```

Comment:
-ST1000: ignore missing package comments
-ST1005: ignore capitalized error messages

Reply:
Thanks! This looks like great for the initial list!

We'd better address ST1005 and remove "-ST1005" in due course.

https://staticcheck.dev/docs/checks/#ST1005